srcpkgs/ansible-core/template

```diff
@@ -1,36 +1,36 @@
 # Template file for 'ansible-core'
 pkgname=ansible-core
-version=2.17.5
-revision=2
+version=2.19.0
+revision=1
+build_style=python3-pep517
 hostmakedepends="python3-setuptools python3-wheel python3-packaging
  python3-straight.plugin python3-docutils python3-Jinja2 python3-yaml"
 depends="python3-cryptography python3-Jinja2 python3-paramiko python3-yaml
  python3-packaging python3-resolvelib python3-pytz git"
-checkdepends="${depends} python3-pytest python3-pytest-xdist
- python3-pytest-forked unzip openssh python3-pytest-mock libselinux
- python3-passlib"
+checkdepends="${depends} openssh python3-bcrypt python3-botocore python3-passlib
+ python3-pexpect python3-pytest python3-pytest-mock python3-pytest-xdist
+ python3-pytest-forked python3-pywinrm python3-voluptuous"
 short_desc="Simple deployment, configuration management and execution framework"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Komeil Parseh <komeilparseh@disroot.org>"
 license="GPL-3.0-or-later"
-homepage="https://www.ansible.com/"
+homepage="https://www.ansible.com"
 changelog="https://raw.githubusercontent.com/ansible/ansible/stable-${version%.*}/changelogs/CHANGELOG-v${version%.*}.rst"
-distfiles="${PYPI_SITE}/a/ansible-core/ansible_core-${version}.tar.gz"
-checksum=ae7f51fd13dc9d57c9bcd43ef23f9c255ca8f18f4b5c0011a4f9b724d92c5a8e
+distfiles="https://github.com/ansible/ansible/archive/refs/tags/v${version}.tar.gz"
+checksum=9a4a4a522c4d06295743f028e210f609faeafa0d6bed3b02857f0dbc3c79a866
```

Comment:
Files from `${PYPI_SITE}` are always preferred to GitHub archives, unless there is a compelling reason to use the latter.